prospector/profiles/profile.py

```diff
@@ -220,12 +220,12 @@ def _determine_strictness(profile_dict, inherits):
     return ('strictness_%s' % strictness), True
 
 
-def _determine_pep8(profile_dict):
-    pep8 = profile_dict.get('pep8', {})
-    if pep8.get('full', False):
-        return 'full_pep8', True
-    elif pep8.get('none', False):
-        return 'no_pep8', True
+def _determine_pycodestyle(profile_dict):
+    pycodestyle = profile_dict.get('pycodestyle', {})
+    if pycodestyle.get('full', False):
```

Comment:
We could update it like `pycodestyle = profile_dict.get('pycodestyle', profile_dict.get('pep8', {}))`